### ISSUE TITLE
FragPipe-PDV viewer of intact glycopeptide

### ISSUE THREAD

**Issue by ming-bi** [Nesvilab/FragPipe#1958]
**- Describe the issue or question:**
I applied fragpipe to searching N-glycopeptide and the results were opened in PDV viewer. But I am confused about the sepectrum. Taking the followed pciture as an example, the last fragment ion was modified six HexNAc. What is the meaning of this fragment ion?
![image](https://github.com/user-attachments/assets/9323ca97-03c1-42dd-8442-2fd0b078ceff)

**Comment by KaiLiCn** [comment]
Hi,

Please use the NIH Glycan symbol as the reference https://www.ncbi.nlm.nih.gov/glycans/snfg.html. The structure is just for the visualization, not the real structure. 

Kai

**Comment by ming-bi** [comment]
thank you for  your answer.
The last fragment ion is it ture? This ion represents a Y ion of glycopeptide. Is it correctly matched  the compositon?

**Comment by KaiLiCn** [comment]
Yes. The last peak was matched to a charge +2 Y ions with the composition shown in the spectrum. You could see the peak m/z putting the cursor on the peak. Please let us know if the mass is incorrect. The mass error should be within your searching tolerance.
